simple-chat-back/src/models/chatroomModel.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+const chatroomSchema = new mongoose.Schema({
+    roomname: {
+        type: String,
+        required: true,
+        trim: true
+      },
+    username: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "User",
+        required: true,
+        trim: true
+    },
+    text: {
+        type: mongoose.Schema.Types.ObjectId,
+        ref: "Text",
+        required: true,
+        trim: true
+      }
+})
+
+const Chatroom = mongoose.model('Chatroom', chatroomSchema )
+module.exports = Chatroom
```

Comment:
add an empty newline. As a rule of thumb, all files should always end with an empty newline